delta/ml/predict.py

```diff
@@ -333,12 +360,19 @@ def _abort(self):
             self._prob_image.abort()
         if self._error_image is not None:
             self._error_image.abort()
+        # DONE: need to add prob_error_image
+        if self._continuous_error_image is not None:
+            self._continuous_error_image.abort()
+        # DONE: need to add prob_error_image
+        if self._continuous_abs_error_image is not None:
+            self._continuous_abs_error_image.abort()
 
     def _process_block(self, pred_image, x, y, labels, label_nodata):
         # DONE: code refactoring
         # create a masked array. The mask is true where pred_image = np.nan
-        pred_image_ma = np.ma.masked_invalid(pred_image)
-        if len(pred_image.shape) == 3:
+        # TODO: why do I have to use squeeze here? When running tests an extra dimension get tacked on/
+        pred_image_ma = np.ma.masked_invalid(np.squeeze(pred_image))
```

Comment:
@bcoltin The extra numpy squeeze because pred_image has a dimension of (10,10,1) when running test_predict_continuous_error_output and test_predict_continuous_error_abs_output.

Reply:
I think we may not handle this case the best way, but we do want to be able to handle images with a single band. I guess they are becoming (10, 10, 1) instead of (10, 10)? I've struggled in other places as well in finding the best way to deal with this.

Reply:
So (10,10,1) as the prediction only shows up when I'm running my tests which call LabelPredictor directly and not through all the image processing code. When I run an actual image from command line the predicted image doesn't have that extra dimension so at least at this point I'm not worried about it.